### PR TITLE
refactor(docker): restart authentik via include

### LINKED ISSUE #888: 💥 [DevOps] Authentik: Change script to have sql dump
## 💥 DevOps

As reviewer I want to see the changes on the database in a file and not a a binary file.
- Application and provider export script
- Flow and stages export script
- System export script

## 🤖 ToDo
- [ ] Change binary dump to sql dump
  - [ ] Application and provider
  - [ ] Flow and stages
  - [ ] System export
- [ ] Add .env to add local, stage or live url

### PR DESCRIPTION
**Pull request by roschaefer**
Motivation
----------
Today I learned: Docker compose v2 has
[include](https://docs.docker.com/compose/multiple-compose-files/include/).

How to test
-----------
1. `docker compose up` from the root folder
2. All services, including authentik, start

### CODE REVIEW

File: authentik/docker-compose.test.yml [hunk 1]
```diff
@@ -1,22 +1,21 @@
 x-mail-settings: &x-mail-settings
   environment:
-    - AUTHENTIK_EMAIL__HOST=mailserver
```
Comment (roschaefer) [inline authentik/docker-compose.test.yml:3]: Cosmetic changes.

File: authentik/docker-compose.test.yml [hunk 2]
```diff
@@ -1,22 +1,21 @@
 x-mail-settings: &x-mail-settings
```
Comment (roschaefer) [inline authentik/docker-compose.test.yml:1]: @mahula this file had executable file permissions, I've `chmod -x` it.

**Review by Elweyn — APPROVED** [review]
Works for me with ``docker compose up --build``

**Comment by Mogge** [comment]
Just one detail. I had differences in the package-lock.json to master after running docker compose up in the main folder

File: authentik/docker-compose.yml
Comment (mahula) [inline authentik/docker-compose.yml:1]: Why the name changes?
The default container names are:

```
NAMES
authentik-server-1
authentik-worker-1
authentik-postgresql-1
authentik-redis-1
```

What is wrong with them in the context of this pull request?

Comment (roschaefer) [reply authentik/docker-compose.yml:1]: Wouldn't it be `server`? How do you prefix the container names?

Comment (roschaefer) [reply authentik/docker-compose.yml:1]: We can rename it, but as you guessed, it's for disambiguation.

**Pull request merged**